tests/datavolume_test.go

```diff
@@ -1447,7 +1454,7 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 			}
 		})
 
-		It("Should create a new scratch PVC when PVC is deleted during import", func() {
+		PIt("Should create a new scratch PVC when PVC is deleted during import", func() {
```

Comment:
The goal is to eventually not need scratch space anymore. However there are still some flows that require scratch space (like importing from registry). So we need this test to work. However the flow it uses to test this, no longer uses scratch space. So maybe you can modify this test to use:
```go
dv := utils.NewDataVolumeWithRegistryImport(dvName, "500Mi", tinyCoreIsoRegistryURL())
```
instead? Not sure if that will work, but worth a try.

Reply:
I tried this suggestion but the importer is failing because of the registry certificate:
```
I0108 12:03:09.541298       1 importer.go:52] Starting importer
I0108 12:03:09.541366       1 importer.go:122] begin import process
I0108 12:03:09.541376       1 data-processor.go:350] Calculating available size
I0108 12:03:09.541382       1 data-processor.go:362] Checking out file system volume size.
I0108 12:03:09.541385       1 data-processor.go:370] Request image size not empty.
I0108 12:03:09.541391       1 data-processor.go:375] Target size 500Mi.
I0108 12:03:09.541419       1 data-processor.go:242] New phase: TransferScratch
I0108 12:03:09.541430       1 registry-datasource.go:83] Copying registry image to scratch space.
I0108 12:03:09.541433       1 transport.go:187] Downloading image from 'docker://cdi-docker-registry-host.cdi/tinycoreqcow2', copying file from 'disk' to '/scratch'
E0108 12:03:09.560972       1 transport.go:74] Could not create image reference: error pinging docker registry cdi-docker-registry-host.cdi: Get https://cdi-docker-registry-host.cdi/v2/: x509: certificate signed by unknown authority
E0108 12:03:09.560995       1 data-processor.go:239] Get https://cdi-docker-registry-host.cdi/v2/: x509: certificate signed by unknown authority
error pinging docker registry cdi-docker-registry-host.cdi
```
I need to check what is going wrong

Reply:
Check https://github.com/kubevirt/containerized-data-importer/blob/master/tests/datavolume_test.go#L98-L104 to see an example of how to setup the certificate (or maybe just call the function itself)

Reply:
Wit the right function and certificate, it works